examples/vts-react-test/src/App.js

```diff
@@ -43,6 +43,9 @@ function App() {
       setModelName(modelName)
     })
 
+    return () => {
+      apiClient.disconnect()
```

Comment:
This ensures we close the previous connection when the component unmounts. I faced a similar issue in my own plugin, which prompted me to open this PR.